modules/mri_violations/php/mri_protocol_violations.class.inc

```diff
@@ -20,7 +20,7 @@ namespace LORIS\mri_violations;
  *
  * @category MRI
  * @package  Main
- * @author   Zia Mohadesz <zia.mohades@gmail.com>
+ * @author   Zia Mohades <zia.mohades@gmail.com>
```

Comment:
I don't think you should change other people's names..

Reply:
It's a typo though?

Reply:
@mohadesz sometimes spells his name with a z.

Reply:
Ah i thought the username was mohadesz[ia].

Reply:
anyway I don't really care about this